<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNanoVolt</class>
 <widget class="QWidget" name="WidgetNanoVolt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QCheckBox" name="check_lsync">
     <property name="text">
      <string>Line cycle synchronization</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="btn_connect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="12">
    <widget class="QListWidget" name="list_port"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_port">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="combo_range">
     <item>
      <property name="text">
       <string notr="true">10 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">100 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1 V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">10 V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">100 V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Autoscale</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="_label_4">
     <property name="text">
      <string>Rate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spin_rate">
     <property name="suffix">
      <string> PLC</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="check_analog">
     <property name="text">
      <string>Analog filter</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="_label_3">
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combo_channel">
     <item>
      <property name="text">
       <string>Channel 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_label_1">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="_label_2">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Digital filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="_label_5">
        <property name="text">
         <string>Average</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="_label_6">
        <property name="text">
         <string>Filter count</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spin_filter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="check_digital">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radio_repeating">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Repeating</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radio_moving">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Moving</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_channel</tabstop>
  <tabstop>combo_range</tabstop>
  <tabstop>spin_rate</tabstop>
  <tabstop>check_lsync</tabstop>
  <tabstop>check_analog</tabstop>
  <tabstop>check_digital</tabstop>
  <tabstop>radio_moving</tabstop>
  <tabstop>radio_repeating</tabstop>
  <tabstop>spin_filter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>check_digital</sender>
   <signal>toggled(bool)</signal>
   <receiver>radio_repeating</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_digital</sender>
   <signal>toggled(bool)</signal>
   <receiver>radio_moving</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_digital</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_filter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
